commands/remind.js

```diff
@@ -0,0 +1,388 @@
+const Discord = require('discord.js');
+
+const config = require('../config.json');
+
+const errors = require('../helpers/remindErrors.js');
+const remindUtils = require('../utils/remindUtils.js');
+
+const Reminder = require('../databaseFiles/remindersTable.js');
+const SpellChecker = require('spellchecker'); // Used to automatically fix the typos.
+
+const monthsData = require('../data/monthsData.js');
+
+
+module.exports.execute = async (client, message, args) => {
+	// Restrict command usage to accountability-station and command-center channels.
+	if (!(message.channel.id === config.channels.accountability || message.channel.id === config.channels.commandcenter)) {
+		return await message.channel.send(
+			`Whoops, sorry, but the "remind" command is only available in <#${config.channels.accountability}> and <#${config.channels.commandcenter}>.`
+		);
+	}
+
+	if (args.length === 0 || args.length === 1 && (args[0] === 'help' || args[0] === 'info')) {
+		const remindHelp = new Discord.RichEmbed()
+			.setColor('#FFEC09')
+			.setTitle(`${config.emotes.reminders} Knights of Academia Remind Help ${config.emotes.reminders}`)
+			.setDescription('Here are some commands to help you out with reminders!')
+			.addField('Add a reminder',
+				`\`!remind [me to] <task> in <how many> minutes/hours/days/months\`
+				Example: \`!remind me to do laundry in 2 hours\`
```

Comment:
It might be easier to have these separated as fields.

Reply:
As discussed on Discord we can leave it like that for now and split it later down the line if more options are added. ^^